BentoKit/BentoKit/Components/Toggle.swift

```diff
@@ -40,19 +40,21 @@ extension Component {
 
             self.heightComputer = { width, inheritedMargins in
                 let verticalMargins = styleSheet.layoutMargins.verticalTotal
-                let imageWidth: CGFloat = image != nil
-                    ? Dimensions.imageViewWidth + Dimensions.horizontalElementsSpacing
-                    : 0.0
+                let imageWidth: CGFloat = image?.size.width ?? 0
```

Comment:
Between the image and the label, the stack view spacing applies when the UIImageView is not hidden.

Reply:
Thanks for pointing that out, wasn't aware @andersio. Will fix that asap